edx_rest_framework_extensions/middleware.py

```diff
@@ -13,6 +13,7 @@ class RequestMetricsMiddleware(object):
             jwt, bearer, other-token-type, or session-or-unknown
         request_user_agent: The user agent string from the request header.
         request_client_name: The client name from edx-rest-api-client calls.
+        request_user_id: The user id of the request user.
```

Comment:
Nit: Looks like these were already out of alphabetical order, but feel free to order them. :)